MANIFEST.in

```diff
@@ -9,6 +9,7 @@ prune tests
 prune .github
 prune .tox
 
+exclude src/torchpme/_version.py
```

Comment:
are you sure about this? I would think we need this file included in the sdist so we have the same version when building the wheels from the sdist as when building wheels from the git checkout.

Reply:
No I was not. But I tried and the `_version.py` is in the wheel and for the sdist it will be created again. So seems to be okay to have this here.

Reply:
Ahh you might be right with the sdist.

Reply:
I added an ignore for `check-manifest`. This fixes the issue and we have a `_version.py` in the sdist.